src/templates/_partials/_blog_post.twig

```diff
@@ -0,0 +1,16 @@
+<div class="pt-8 border-b border-gray-300 first:pt-0">
+  <p class="text-base leading-[19px] py-2 text-gray-700 font-semibold">
+    {{ blog.date }}
+  </p>
+  <h3 class="text-blue-700 h3">
+    {{ blog.title }}
+  </h3>
```

Comment:
Use `h4` instead of `h3` and wrap it with a anchor tag that links to the same link as `Learn More`